Cargo.toml

```diff
@@ -29,7 +29,10 @@ url = "1"
 humansize = "1"
 # used in date format filter
 chrono = "0.4"
+# used in instrument timing
+stopwatch = "^0.0.7"
+log = "^0.3.8"
```

Comment:
was logging used only when debugging? can it be removed?

Reply:
Removed stopwatch dependency. I do use `log!` by creating a macro called `out!` that points to `log!` for dev but can switch to `println!` if desire is to see logs from testing, since testing does not support.

I can remove this logging if you'd like, but maybe best to leave until you have time to review, in case need to review code more generally and logging might help. Whichever you prefer.

Reply:
I believe I effectively did a rebase. My first and my first pull request, so I may have screwed it up. I hope it helps you diff more easily - but I'm not seeing how it helps so maybe I missed a step.